components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/token/OAuth2TokenEndpoint.java

```diff
@@ -133,7 +139,11 @@ protected Response issueAccessToken(HttpServletRequest request, Map<String, List
             OAuthSystemException, InvalidRequestParentException {
 
         try {
-            startSuperTenantFlow();
+            if (IdentityTenantUtil.isTenantQualifiedUrlsEnabled()) {
+                startTenantFlow();
```

Comment:
The above comment is applied here as well

Reply:
Fixed